doc/changelog.md

```diff
@@ -2,8 +2,26 @@
 
 <h3>New features</h3>
 
-* Catalyst programs can now be used inside of a larger JAX workflow which uses
-  JIT compilation, automatic differentiation, and other JAX transforms.
+* Add a preliminary support for native quantum Adjoint operation. `catalyst.adjoint` computes the

```

Comment:
@grwlf don't forget to merge in `main`, and move this to the newer version in the changelog :)

Reply:
Done, thanks @josh146